app/models/manageiq/providers/base_manager/refresher.rb

```diff
@@ -16,7 +16,7 @@ def initialize(targets)
 
       def options
         return @options if defined?(@options)
-        @options = VMDB::Config.new("vmdb").config[:ems_refresh]
+        @options = Settings.ems_refresh
```

Comment:
I greped all `refresh_parser.rb` and `refresher.rb` for `options` and non, excpet amanzon, was using some. This is also plausible from [settings.yml](https://github.com/ManageIQ/manageiq/blob/faec76397e8e77d65e98432d8d46e38357632832/config/settings.yml#L105-L107) where only options for ec2 are defined.
So, yeah, I guess it does not affect the others :smile: